roles/openshift_hosted_logging/tasks/deploy_logging.yaml

```diff
@@ -77,9 +76,14 @@
   - name: "Process support template"
     shell:  "{{ openshift.common.client_binary }} --config={{ mktemp.stdout }}/admin.kubeconfig process logging-support-template |  {{ openshift.common.client_binary }} --config={{ mktemp.stdout }}/admin.kubeconfig create -f -"
```

Comment:
IIUC this is not needed now because the deployer runs it. https://github.com/openshift/origin-aggregated-logging/blob/master/deployer/scripts/install.sh#L113